### DISCUSSION TITLE
Multiple Accounts

### DISCUSSION THREAD

**Discussion by Questinghero** [unknownskl/xbox-xcloud-client#487] [Ideas]
I would love to be able to setup multiple accounts in this app. I have a shared Steam Deck for my family, with several Steam accounts setup on it. I would love to be able to have the app save several xbox logins, so I could just pass something like --xbox-user=XboxUserName in the command line arguments and the app would launch with that user account.

another option would be to specify where the profile is saved, so I could just have the shortcut on each steam user point to a different profile --save-path=~/Application/2nduser or something like that.

Thanks,

**Comment by emvystudio** [comment]
Would also love this! Was just posting the same thing and did a search. I have multiple users and one steam deck, would love to streamline the login.
